modules/vue/tests/playwright/ModulesOrder.spec.js

```diff
@@ -0,0 +1,220 @@
+import { test, expect } from '@playwright/test'
+import { makeProject } from '@storefront-x/testing'
+
+test('modules order', async ({ page }) => {
```

Comment:
Why are these tests in the vue module? Move them either to core module or to base module.